APIv2 tests: python something != something-podman

Seeing this occasionally since Dec 30:
```
________________________ ContainerTestCase.test_attach _________________________
         
         self = <python.rest_api.test_v2_0_0_container.ContainerTestCase testMethod=test_attach>
         
             def test_attach(self):
                 r = requests.post(
                     self.podman_url + "/v1.40/containers/create?name=topcontainer",
                     json={"Cmd": ["sh", "-c", "echo podman; sleep 100"], "Image": "alpine:latest"},
                 )
                 self.assertEqual(r.status_code, 201, r.text)
                 payload = r.json()
             
                 r = requests.post(
                     self.podman_url
                     + f"/v1.40/containers/{payload['Id']}/start"
                 )
                 self.assertEqual(r.status_code, 204, r.text)
             
                 r = requests.post(
                     self.podman_url
                     + f"/v1.40/containers/{payload['Id']}/attach?logs=true&stream=false"
                 )
                 self.assertIn(r.status_code, (101, 200), r.text)
                 # see the attach format docs, stdout = 1, length = 7, message = podman\n
         >       self.assertEqual(r.content, b"\x01\x00\x00\x00\x00\x00\x00\x07podman\n", r.text)
         E       AssertionError: b'' != b'\x01\x00\x00\x00\x00\x00\x00\x07podman\n' :
         
         test/apiv2/python/rest_api/test_v2_0_0_container.py:155: AssertionError
         =========================== short test summary info ============================
         FAILED test/apiv2/python/rest_api/test_v2_0_0_container.py::ContainerTestCase::test_attach
         =================== 1 failed, 43 passed in 78.37s (0:01:18) ====================
```
If I read the URL correctly, this has something to do with `logs`, and ISTR other flakes in podman-remote logs but can't find the issue number.

* fedora-37 : APIv2 test on fedora-37 (root)
  * PR #17165
    * [01-19 06:53](https://api.cirrus-ci.com/v1/task/5030036521091072/logs/main.log)
  * PR #16997
    * [01-05 03:10](https://api.cirrus-ci.com/v1/task/4694463075844096/logs/main.log)
  * PR #16978
    * [01-04 03:14](https://api.cirrus-ci.com/v1/task/4749243169112064/logs/main.log)
  * PR #16810
    * [12-30 10:43](https://api.cirrus-ci.com/v1/task/5556295575535616/logs/main.log)
    * [12-30 10:30](https://api.cirrus-ci.com/v1/task/5214991905718272/logs/main.log)

--- a/test/apiv2/python/rest_api/test_v2_0_0_container.py
+++ b/test/apiv2/python/rest_api/test_v2_0_0_container.py
@@ -146,6 +146,19 @@ class ContainerTestCase(APITestCase):
         )
         self.assertEqual(r.status_code, 204, r.text)
 
+        # wait for the log message to appear to avoid flakes on slow systems
+        # with the /attach?logs=true test below
+        for _ in range(5):
+            r = requests.get(
+                self.podman_url
+                + f"/v1.40/containers/{payload['Id']}/logs?stdout=true"
+            )
+            self.assertIn(r.status_code, (101, 200), r.text)
+            if r.content == b"\x01\x00\x00\x00\x00\x00\x00\x07podman\n":
+                break
+
+            time.sleep(1)
+
         r = requests.post(
             self.podman_url
             + f"/v1.40/containers/{payload['Id']}/attach?logs=true&stream=false"

